.github/workflows/omni.yml

```diff
@@ -404,7 +411,7 @@ jobs:
     - name: Test
       uses: burrunan/gradle-cache-action@v1
       with:
-        arguments: --no-parallel --no-daemon --scan jandex test jacocoReport
+        arguments: --no-parallel --no-daemon --scan jandex test jacocoReport -PskipJavadoc
```

Comment:
@sehrope , I don't understand how `omni.yml` generator works, so I skip javadoc for all omni jobs.